api/HTMLOptionsCollection.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLOptionsCollection",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "25"
```

Comment:
I have the same doubt here as in https://github.com/mdn/browser-compat-data/pull/7762#discussion_r542636041. I think this was supported earlier.